hardware/AMEL/samd/variants/AMEL_SmartEverything/variant.h

```diff
@@ -240,6 +240,11 @@ void setStepUp(uint32_t on);
 */
 bool isOnBattery(void);
 
+
+void gpsForceOn(void);
+void sfxSleep(void);
+void sfxWakeup(void);
+
```

Comment:
The idea is to have all the libraries independent from the ASME,
Since are UART/I2C based, they could be reused from any other Arduino product that want to use the same chip.
But about the SFX powerSafe feature, the pin is directly connected to an internal IoExtender, so in this case if pure ASME and not reusable, the library will call this function that the other unit with Telit chip has to redefine. 

Same for the Telit GPS